Code/Source/URDF/FbxGenerator.cpp

```diff
@@ -0,0 +1,425 @@
+            m_basicNodes.clear();
+
+            m_basicNodes.push_back(GetFbxHeaderExtension());
+            m_basicNodes.push_back(GetGlobalSettings());
+            m_basicNodes.push_back(GetDocuments());
+            m_basicNodes.push_back(Node("References"));
+            m_basicNodes.push_back(GetDefinitions());
+            m_basicNodes.push_back(*m_objects);
+            m_basicNodes.push_back(GenerateConnections());
+            m_basicNodes.push_back(Node("Takes"));
+
+            m_nodesUpdated = true;
+        }
+
+        Node FbxGenerator::GetFbxHeaderExtension() const
+        {
+            Node fbxHeader("FBXHeaderExtension");
+            fbxHeader.AddChild("FBXHeaderVersion", 1003);
+            fbxHeader.AddChild("FBXVersion", 7500);
+            fbxHeader.AddChild(GetTimeStamp());
+            fbxHeader.AddChild("Creator", "O3DE ROS2 Gem");
+            fbxHeader.AddChild(GetSceneInfo());
+
+            return fbxHeader;
+        }
+
+        Node FbxGenerator::GetGlobalSettings() const
+        {
+            Node globalSettings("GlobalSettings");
+            globalSettings.AddChild("Version", 1000);
+
+            Node properties("Properties70");
+            properties.AddChild(Node("P", {"UpAxis", "int", "Integer", "", 1}));
+            properties.AddChild(Node("P", {"UpAxisSign", "int", "Integer", "", 1}));
+            properties.AddChild(Node("P", {"FrontAxis", "int", "Integer", "", 2}));
+            properties.AddChild(Node("P", {"FrontAxisSign", "int", "Integer", "", 1}));
+            properties.AddChild(Node("P", {"CoordAxis", "int", "Integer", "", 0}));
+            properties.AddChild(Node("P", {"CoordAxisSign", "int", "Integer", "", 1}));
+            properties.AddChild(Node("P", {"OriginalUpAxis", "int", "Integer", "", 1}));
+            properties.AddChild(Node("P", {"OriginalUpAxisSign", "int", "Integer", "", 1}));
+            properties.AddChild(Node("P", {"UnitScaleFactor", "double", "Number", "", 1}));
+            properties.AddChild(Node("P", {"OriginalUnitScaleFactor", "double", "Number", "", 1}));
+            properties.AddChild(Node("P", {"AmbientColor", "ColorRGB", "Color", "", 0, 0, 0}));
+            properties.AddChild(Node("P", {"DefaultCamera", "KString", "", "", "Producer Perspective"}));
+            properties.AddChild(Node("P", {"TimeMode", "enum", "", "", 11}));
+            properties.AddChild(Node("P", {"TimeProtocol", "enum", "", "", 2}));
+            properties.AddChild(Node("P", {"SnapOnFrameMode", "enum", "", "", 0}));
+            properties.AddChild(Node("P", {"TimeSpanStart", "KTime", "Time", "", 1924423250}));
+            properties.AddChild(Node("P", {"TimeSpanStop", "KTime", "Time", "", 1924423250}));
+            properties.AddChild(Node("P", {"CustomFrameRate", "double", "Number", "", -1}));
+            properties.AddChild(Node("P", {"TimeMarker", "Compound", "", ""}));
+            properties.AddChild(Node("P", {"CurrentTimeMarker", "int", "Integer", "", -1}));
```

Comment:
IMHO moving this to a config file will complicate the implementation (because then some kind of nodes reading method is necessary, for example, FBX reader). Also, this is rather a constant part that shouldn't be changed by us.

But if you think it's necessary, then I would propose creating a default/config FBX file only with this header which will be loaded and used in generated files. I thought about JSON but then the additional mapping between FBX constant parameters and JSON parameters is necessary (I'm afraid it could be confusing because of the number of parameters). Anyway, I think the FBX reader is not trivial (even if the structure is created), so it could take some time.

So, what do you think?